### PR TITLE
MVP fixes for #16

### PR DESCRIPTION
**Pull request by lhao03**
based on comments in #16

### CODE REVIEW

**Comment by lhao03** [comment]
how did you get graph explorer to show in a different language?

**Comment by ayubun** [comment]
> how did you get graph explorer to show in a different language?

I just added the locale in the URL; so localhost:3000 became localhost:3000/zh-CN

File: src/app/views/query-runner/request/permissions/Permission.tsx [hunk 1]
```diff
@@ -40,7 +40,9 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
   }
 
   public componentDidUpdate = (prevProps: IPermissionProps) => {
-    if ((prevProps.sample !== this.props.sample) || (prevProps.permissionsPanelOpen !== this.props.permissionsPanelOpen) || (prevProps.permissionModeType !== this.props.permissionModeType)) {
+    if (prevProps.sample !== this.props.sample
+      || prevProps.permissionsPanelOpen !== this.props.permissionsPanelOpen
+      || prevProps.permissionModeType !== this.props.permissionModeType) {
```
Comment (ayubun) [inline src/app/views/query-runner/request/permissions/Permission.tsx]:
```suggestion
        || prevProps.permissionsPanelOpen !== this.props.permissionsPanelOpen
        || prevProps.permissionModeType !== this.props.permissionModeType) {
```
The extra indentation is an intentional formatting for if statements / functions of the like

**Review by ayubun — CHANGES_REQUESTED** [review]

**Review by ayubun — APPROVED** [review]

**Comment by ayubun** [comment]
LGTM; We should probably merge sooner rather than later so we can continue off on #16 where Changsheng was commenting

**Pull request merged**